packages/vite-plugin-cloudflare/src/index.ts

```diff
@@ -100,6 +135,11 @@ export function cloudflare<
 
 					return {
 						...workerOptions,
+						compatibilityFlags: [
+							...(workerOptions.compatibilityFlags ?? []),
+							'nodejs_compat',
```

Comment:
OOC - why do we need to add this flag?

Reply:
```suggestion
							...(workerOptions.compatibilityFlags?.includes('nodejs_compat')
								? ['nodejs_compat']
								: []),
```

Reply:
oops, no sorry I added this by mistake, thanks for spotting this! 🙇

Reply:
(by the way, it was when I was investigating this: https://github.com/flarelabs-net/vite-plugin-cloudflare/pull/15/files#diff-544c6cfcc950af42d4e59824927b6ba7dd94705534aa7df3ea200f339b3c9d3eR8-R15)

Reply:
removed 👍